website/docs/r/advanced_cluster.html.markdown

```diff
@@ -246,6 +246,7 @@ replication_specs {
 
```

Comment:
In clusters we include the following note at the top of the docs page. We should do the same here.  https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/r/cluster.html.markdown?plain=1#L15